source/preprocessor.tex

```diff
@@ -1292,11 +1292,14 @@
 
 \indextext{__CPLUSPLUS@\xname{cplusplus}}%
 \item \xname{cplusplus}\\
-The integer literal \tcode{\cppver}.\footnote{It is intended that future
+The integer literal \tcode{\cppver}.
+\begin{note}
+It is intended that future
 versions of this International Standard will
 replace the value of this macro with a greater value.
 Non-conforming compilers should use a value with at most
```

Comment:
The Directives have a similar ban against "shall"/"should" in notes too...

Reply:
Yes, this is still not allowed by ISO. How about "Non-conforming compilers can use a value ..."?

Or we could drop that part of the note, since it no longer reflects reality. Before C++17 was close to publication GCC used `201500` to indicate "more than C++14 but less than C++1z", and Clang used `201406`.

Reply:
First, we can't really say something about non-conforming compilers; they're non-conforming anyway. Second, we now have feature-test macros.
So, the update removes the "should" sentence talking about non-conforming compilers.